src/components/greetings.jsx

```diff
@@ -0,0 +1,27 @@
+import { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import fetchGreeting from '../redux/actions';
+
+function Greeting({ greeting, fetchGreeting }) {
```

Comment:
- [ ] [REQUIRED] You passed your `greeting` as a prop to this component where you are supposed to get the greeting from your redux store. Kindly use useSelector to get your greetings from the redux store because currently this greeting is undefined and the page is blank. Take a look at the console log statement I did, it prints the greeting as undefined.

<img width="1367" alt="Screenshot 2023-11-11 at 20 21 13" src="https://github.com/MasumaJaffery/Hello-React-Frontend/assets/58692590/7a601b01-7fd8-4991-be33-c523445632aa">